### PR TITLE
fix: use first issue without inline on mergeLineIssues on multiplie issues

### PR DESCRIPTION
**Pull request by peakle**
fix for: #3230

### CODE REVIEW

**Comment by ldez** [comment]
@peakle please read the internal team message https://github.com/orgs/golangci/teams/team/discussions/46

**Comment by peakle** [comment]
> offtopic: @peakle please read the internal team message https://github.com/orgs/golangci/teams/team/discussions/46

thank you, didn't see , done

File: pkg/result/processors/fixer.go [hunk 1]
```diff
@@ -136,7 +136,7 @@ func (f Fixer) mergeLineIssues(lineNum int, lineIssues []result.Issue, origFileL
 		r := i.Replacement
 		if r.Inline == nil || len(r.NewLines) != 0 || r.NeedOnlyDelete {
 			f.log.Infof("Line %d has multiple issues but at least one of them isn't inline: %#v", lineNum, lineIssues)
-			return &lineIssues[0]
+			return i
```
Comment (alexandear) [inline pkg/result/processors/fixer.go:139]: Do we need a unit test to cover that case?

Comment (peakle) [reply pkg/result/processors/fixer.go:139]: done

Comment (peakle) [reply pkg/result/processors/fixer.go:139]: one test enough or need more test cases?

**Comment by peakle** [comment]
@ldez , @alexandear  any updates?

**Comment by peakle** [comment]
@ldez ping

**Review by alexandear — APPROVED** [review]

**Review by ldez — APPROVED** [review]
LGTM

Sorry for the extremely late review.

**Pull request merged**